### PR TITLE
Admin - Test - Create Menu

### PR DESCRIPTION
**Pull request by hungbkdn**
Update comment for updatemenuitemtest, remove folder dailymenu in Request folder

### CODE REVIEW

File: tests/Browser/AdminCreateMenuTest.php [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+                    ->waitFor(null, '1')
+                    ->screenshot('AdminCreateMenu-FoodSelected')
+                    ->type('quantity', 5)
+                    ->click('#clear-input')
+                    ->assertInputValue('date', null)
+                    ->assertSelected('#select-food', null)
+                    ->assertInputValue('quantity', null);
+        });
+    }
+}
```
Comment (ghost) [inline tests/Browser/AdminCreateMenuTest.php]: Please add case, select incorrect date and disable **Add** button

**Review by ghost — COMMENTED** [review]
Please check my comment

**Comment by hungbkdn** [comment]
I have updated, please check it again!
Thank you so much @at-vietpham

**Comment by ghost** [comment]
LGTM

**Pull request merged**